packages/dev/s2-docs/src/VisualExampleClient.tsx

```diff
@@ -12,13 +12,19 @@ import {getColorChannels, parseColor} from 'react-stately';
 import {ListBox, ListBoxItem, Size} from 'react-aria-components';
 import {mergeStyles} from '../../../@react-spectrum/s2/style/runtime';
 import type {PropControl} from './VisualExample';
-import React, {createContext, Fragment, isValidElement, lazy, ReactNode, Ref, Suspense, useContext, useEffect, useMemo, useRef, useState} from 'react';
+import React, {createContext, createElement, Fragment, isValidElement, lazy, ReactNode, Ref, Suspense, useContext, useEffect, useMemo, useRef, useState} from 'react';
 import RemoveCircle from '@react-spectrum/s2/icons/RemoveCircle';
 import {TabLink} from './FileTabs';
 import {useLocale} from 'react-aria';
+// eslint-disable-next-line
+import icons from '/packages/@react-spectrum/s2/s2wf-icons/*.svg';
```

Comment:
The problem with this is that it breaks the code splitting, so now every icon will be loaded on every page even if the component doesn't have an icon picker...

Reply:
Ah, hmm is there a way to send it as a rsc?

Reply:
Otherwise I guess I could send the svg directly

Reply:
You could maybe load it dynamically if you export the icon map from the icon picker file?

```tsx
const LazyIcon = lazy(() => import('./IconPicker')
  .then(({icons}) => ({default: ({icon}) => createElement(icons[icon])}))
```

essentially this creates a `LazyIcon` component that accepts an `icon` prop and gets that icon from the map.

```tsx
<LazyIcon icon="whatever" />
```